src/iris/api.py

```diff
@@ -181,23 +181,30 @@ def ts_to_sql_datetime(ts):
 }
 
 incident_query = '''SELECT %s FROM `incident`
+LEFT JOIN `incident_claim` ON `incident_claim`.`incident_id` = `incident`.`id` AND `incident_claim`.`id` =
+          (SELECT `id` FROM `incident_claim` WHERE `incident_claim`.`incident_id` = `incident`.`id` ORDER BY `time` DESC, `id` DESC LIMIT 1)
```

Comment:
The problem I ran into with using MAX(`time`) is that it is quite common for there to be several incident_claim actions in the same second. Because of that, I think ordering on `id` is safer.